src/caffe/test/test_triplet_loss_layer

```diff
@@ -0,0 +1,125 @@
+#include <algorithm>
+#include <cmath>
+#include <cstdlib>
+#include <cstring>
+#include <vector>
+
+#include "gtest/gtest.h"
+
+#include "caffe/blob.hpp"
+#include "caffe/common.hpp"
+#include "caffe/filler.hpp"
+#include "caffe/vision_layers.hpp"
+
```

Comment:
1. Missing .cpp file extension
2. Is '#include "caffe/vision_layers.hpp"' needed?
3. Missing '#include "caffe/layers/triplet_loss_layer.hpp"'